index.html

```diff
@@ -458,6 +467,142 @@ <h4>
           </dl>
         </section>
       </section>
+      <section>
+        <h2>
+          Showing the payment handler
+        </h2>
+        <p>
+          The steps for showing the Basic Card payment handler are given by the
+          following algorithm. These steps run when a payment UI is first
+          presented to the end-user, as a successful call to a
+          <a>PaymentRequest</a>'s <a data-cite=
+          "payment-request#dom-paymentrequest-show"><code>show()</code></a>
+          method. The steps take a <a>PaymentRequest</a> <var>request</var>.
+        </p>
+        <div class="note">
+          <p>
+            In the case where a payment request is shown with multiple
+            applicable <a>PaymentMethodData</a>s, this algorithm selects the
+            last <a data-cite=
+            "payment-request/#dfn-serializedmethoddata">[[\serializedMethodData]]</a>
+            whose PMI is "<code>basic-card</code>". (i.e., "the last one
+            wins").
+          </p>
+          <pre class="example" title="Last one wins">
+            const methodData = [
+            // This one is ignored.
+            {
+              supportedMethods: "basic-card",
+              data: {
+                supportedNetworks: ["visa", "mastercard"],
+              },
+            },
+            // ✅ This second one gets used!
+            {
+              supportedMethods: "basic-card",
+              data: {
+                supportedNetworks: ["visa"],
+              },
+            },
+            // This one gets skipped, because it's not applicable to
+            // basic card.
+            {
+              supportedMethods: "https://example.com/bobpay",
+            },
+          ];
+
+          // See above for which methodData gets picked.
+          await new PaymentRequest(methodData, details, options).show();
+          </pre>
+        </div>
+        <ol class="algorithm">
+          <li>For each <a data-cite="infra#pair">pair</a> (<a>DOMString</a>
+          <var>id</var> / string <var>data</var>) of
+          <var>request</var>.<a data-cite=
+          "payment-request/#dfn-serializedmethoddata">[[\serializedMethodData]]</a>
+          in reverse order:
+            <ol>
+              <li>If <var>id</var> is not "<a>basic-card</a>", continue.
+              </li>
+              <li>Let <var data-type="object">object</var> be result of
+              <a data-cite="ecma-262#sec-json.parse">JSON-parsing</a>
+              <var>data</var>.
+              </li>
+              <li>If <var>object</var> is <code>null</code>, continue.
```

Comment:
Can `object` also be `null` if the original `PaymentMethodData` didn't contain a `data` field? Should the absence of `data` field also be considered "all networks are supported"?

Reply:
> Can object also be null if the original PaymentMethodData didn't contain a data field?

Yes, the PR API substitutes missing with null. 

> Should the absence of data field also be considered "all networks are supported"?

I don’t feel strongly about it. From way back, we’d agreed that only “[]” would mean “all”. Thus I’m inclined to treat no data as “ignore”.